pilot/pkg/networking/core/v1alpha3/cluster.go

```diff
@@ -119,11 +125,31 @@ func (configgen *ConfigGeneratorImpl) buildOutboundClusters(cb *ClusterBuilder,
 			if port.Protocol == protocol.UDP {
 				continue
 			}
-			lbEndpoints := cb.buildLocalityLbEndpoints(networkView, service, port.Port, nil)
+			clusterName := model.BuildSubsetKey(model.TrafficDirectionOutbound, "", service.Hostname, port.Port)
+			clusterKey := &clusterCache{
```

Comment:
nit: can we move this to a constructor?

Reply:
I would be slightly worried about creating a func with tons of parameters. At this point I think we just need `cb`,, service, and port though so perhaps its ok, will try it out